# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1405604007 14400
#      Thu Jul 17 09:33:27 2014 -0400
# Node ID f1edad3b410619da3b1a14f7b16179089933693b
# Parent  3d25cd89fec06f8da38fdb3695a6fd5e0e3caedd
avoid some unused parameter warnings

* octave-dock-widget.cc: Avoid unused parameter warning.
* profiler.cc: Avoid unused parameter warning.

diff --git a/libgui/src/octave-dock-widget.cc b/libgui/src/octave-dock-widget.cc
--- a/libgui/src/octave-dock-widget.cc
+++ b/libgui/src/octave-dock-widget.cc
@@ -193,17 +193,21 @@ octave_dock_widget::make_window ()
 
 #endif
 
   _floating = true;
 }
 
 // dock the widget
 void
+#if defined (Q_OS_WIN32)
 octave_dock_widget::make_widget (bool dock)
+#else
+octave_dock_widget::make_widget (bool)
+#endif
 {
 #if defined (Q_OS_WIN32)
 
   // windows: Since floating widget has no parent, we have to read it
 
   QSettings *settings = resource_manager::get_settings ();
 
   // save last floating geometry if widget really was floating
diff --git a/libinterp/corefcn/profiler.cc b/libinterp/corefcn/profiler.cc
--- a/libinterp/corefcn/profiler.cc
+++ b/libinterp/corefcn/profiler.cc
@@ -101,17 +101,17 @@ profile_data_accumulator::tree_node::ent
   else
     retval = pos->second;
 
   ++retval->calls;
   return retval;
 }
 
 profile_data_accumulator::tree_node*
-profile_data_accumulator::tree_node::exit (octave_idx_type fcn)
+profile_data_accumulator::tree_node::exit (octave_idx_type /* fcn */)
 {
   // FIXME: These assert statements don't make sense if profile() is called
   //        from within a function hierarchy to begin with.  See bug #39587.
   //  assert (parent);
   //  assert (fcn_id == fcn);
 
   return parent;
 }
